packages/bpk-component-chip-group/src/Nudger.tsx

```diff
@@ -0,0 +1,113 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { type MutableRefObject, useEffect, useState } from 'react';
+
+import { BpkButtonV2, BUTTON_TYPES } from '../../bpk-component-button';
+import { CHIP_TYPES } from '../../bpk-component-chip';
+import { withButtonAlignment } from '../../bpk-component-icon/index';
+import ArrowLeft from '../../bpk-component-icon/sm/long-arrow-left';
+import ArrowRight from '../../bpk-component-icon/sm/long-arrow-right';
+import { cssModules, isRTL } from '../../bpk-react-utils/index';
+
+import type { ChipStyleType } from './BpkChipGroup';
+
+import STYLES from './Nudger.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+
+const CHIP_STYLE_TO_BUTTON_STYLE = {
+  [CHIP_TYPES.default]: BUTTON_TYPES.secondary,
+  [CHIP_TYPES.onDark]: BUTTON_TYPES.secondaryOnDark,
+  [CHIP_TYPES.onImage]: BUTTON_TYPES.primaryOnDark,
+}
+
+
+type Props = {
+  ariaLabel: string;
+  chipStyle?: ChipStyleType;
+  scrollContainerRef: MutableRefObject<HTMLElement | null>;
+  leading?: boolean;
+}
+
+
+const AlignedLeftArrowIcon = withButtonAlignment(ArrowLeft);
+const AlignedRightArrowIcon = withButtonAlignment(ArrowRight);
+
+// Chosen based on feeling good with the example stories
+const SCROLL_DISTANCE = 150;
+
+const Nudger = ({
+  ariaLabel,
+  chipStyle = CHIP_TYPES.default,
+  leading = false,
+  scrollContainerRef
+}: Props) => {
+  const [show, setShow] = useState(false);
+  const [enabled, setEnabled] = useState(true);
+
+  const rtl = isRTL();
+  const isLeft = (leading && !rtl) || (!leading && rtl);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      if (!scrollContainerRef.current) {
+        return;
+      }
+
+      const { offsetWidth, scrollLeft, scrollWidth } = scrollContainerRef.current;
+      const scrollValue = rtl ? -Math.floor(scrollLeft) : Math.ceil(scrollLeft);
+      const showLeading = scrollValue > 0;
+      const showTrailing = scrollValue < scrollWidth - offsetWidth;
+
+      setEnabled((leading && showLeading) || (!leading && showTrailing))
+      setShow(showLeading || showTrailing);
+    }, 100);
+    return () => clearInterval(interval);
+  }, [leading, rtl, scrollContainerRef]);
+
+  const classNames = getClassName(
+    'bpk-chip-group-nudger',
+    `bpk-chip-group-nudger--${leading ? "leading" : "trailing"}`,
+    !show && `bpk-chip-group-nudger--hidden`,
+  )
+
+  return (
+    <div className={classNames}>
+      <BpkButtonV2
+        title={ariaLabel}
+        type={CHIP_STYLE_TO_BUTTON_STYLE[chipStyle]}
+        iconOnly
+        disabled={!show || !enabled}
+        onClick={() => {
+          if (scrollContainerRef.current) {
+            scrollContainerRef.current.scrollBy({
+              left: isLeft ? -SCROLL_DISTANCE : SCROLL_DISTANCE,
```

Comment:
Is this for RTL? Can we use that instead?

Reply:
This is a little complex. `isLeft` is calculated as a combination of `isRtl` and `isLeading`. We need to know if it's left or right to choose the correct icon and to also scroll in the right direction when it's pressed.

So I think we are already doing it as simply as we can unfortuantely